modules/accredited_representative_portal/app/models/accredited_representative_portal/icn_temporary_identifier.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class IcnTemporaryIdentifier < ApplicationRecord
+    EXPIRATION_PERIOD = 60.days.freeze
```

Comment:
I wouldn't have considered freezing an `ActiveSupport::Duration`. It doesn't have any public writers, but it does have ivars.

That's making me consider what the differences are between this and `1` for example.